tt_metal/impl/device/device_pool.cpp

```diff
@@ -356,6 +356,9 @@ std::vector<Device*> DevicePool::get_all_active_devices() const {
 }
 
 bool DevicePool::close_device(chip_id_t device_id) {
+    // Sync and close one device
```

Comment:
This API should never be called when we have more than one device available on a machine? Should we just error out if a user attempts this?

Reply:
You can call it on T3k, when you open a workload on just chip 0 for example.